lstchain/scripts/lstchain_add_source_dependent_parameters.py

```diff
@@ -49,8 +53,26 @@ def main():
             pass
 
     dl1_params = pd.read_hdf(dl1_filename, key=dl1_params_lstcam_key)
-    src_dep_df = get_source_dependent_parameters(dl1_params, config)
-    write_dataframe(src_dep_df, dl1_filename, dl1_params_src_dep_lstcam_key)
+ 
+    src_dep_df_list = get_source_dependent_parameters(dl1_params, config)
+
+    is_simu = (dl1_params['mc_type'] >= 0).all() if 'mc_type' in dl1_params.columns else False
+
+    if is_simu:
+        write_dataframe(src_dep_df_list[0], dl1_filename, dl1_params_src_dep_lstcam_key)
```

Comment:
I think this would be easier to read if the params were in a dict or as separated on/off params

Reply:
same as DL1, I modified to use a dictionary and `MultiIndex`